deploy/modules/share/tidb-cluster-release/data.tf

```diff
@@ -0,0 +1,11 @@
+data "external" "tidb_hostname" {
+  depends_on  = [helm_release.tidb-cluster]
+  working_dir = path.cwd
+  program     = ["bash", "-c", "kubectl --kubeconfig ${var.kubeconfig_filename} get svc -n ${var.cluster_name} ${var.cluster_name}-tidb -o json | jq '.status.loadBalancer.ingress[0]'"]
```

Comment:
Can this read the right kubeconfig file when in a separated module?

Reply:
Yes, the kubeconfig_filename should be the relative path to `path.cwd` by convention, and this is now guaranteed by all the modules we provide